UnicornEngine/video/include/unicorn/video/vulkan/Buffer.hpp

```diff
@@ -0,0 +1,108 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_VIDEO_VULKAN_BUFFER_HPP
+#define UNICORN_VIDEO_VULKAN_BUFFER_HPP
+
+#include <vulkan/vulkan.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+namespace vulkan
+{
+/**
+ * @brief Wrapper for vk::Buffer item in Vulkan API
```

Comment:
It has CopyToBuffer, but of course this function can be in some sort VulkanUtils.hpp and vk::Buffer source and vk::Buffer dest.